benchto-driver/src/main/java/io/trino/benchto/driver/listeners/profiler/async/AsyncProfiler.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.benchto.driver.listeners.profiler.async;
+
+import io.trino.benchto.driver.listeners.profiler.QueryProfiler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Component;
+
+import javax.management.MBeanServerConnection;
+import javax.management.ObjectName;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.stream.Collectors;
+
+@Component
+@ConditionalOnProperty(prefix = "benchmark.feature.profiler.async", name = "enabled", havingValue = "true")
+public class AsyncProfiler
```

Comment:
please document the node/pod requirements like libasyncProfiler.so

Reply:
added entry to `README.me`